intent-classification-v0/ml_modules/bert_classifier.py

```diff
@@ -0,0 +1,71 @@
+from dotenv import load_dotenv
+from typing import Dict, Tuple, Set, Optional, List, Union
+import logging
+import os
+import requests
+
+load_dotenv(".env")
+
+# Add logging.
+logging.basicConfig(level=logging.INFO)
+
+API_URL = "https://api-inference.huggingface.co/models/deprem-ml/bert-fine-tuned-intent"
+API_TOKEN = os.getenv("HF_HUB_TOKEN")
+
+headers = {"Authorization": f"Bearer {API_TOKEN}"}
+
+# The score threshold to deem a label as positive.
+CLASSIFICCTION_THRESHOLD = 0.5
+
+
+class BertClassifier(object):
+    """
+    BERT based classifier to uses huggingface inference API to classify tweets.
+
+    Example usage:
+    >>> classifier = BertClassifier()
+    >>> classifier.classify("Kırıkkale'de deprem oldu.")
+    ["KURTARMA"]
+    """
+
+    def __init__(self, classification_threshold=CLASSIFICCTION_THRESHOLD):
+        self.classification_threshold = classification_threshold
+
+    def __query(self, text):
+
+        response = requests.post(
+            API_URL, headers=headers, json={"inputs": text})
+        return response.json()
+
+    def all_intents(self):
+        """Returns list of all possible intents this classifier can classify."""
+        return ["ALAKASIZ", "KURTARMA", "BARINMA", "SAGLIK", "LOJISTIK", "SU", "YAGMA", "YEMEK", "GIYSI", "ELEKTRONIK"]
```

Comment:
Tuvalet is missing here.

Reply:
Model (deprem-ml/bert-fine-tuned-intent) doesn't have TUVALET, just double checked this is the full list of intents it returns rn.